lsif-semanticdb/src/main/java/com/sourcegraph/lsif_semanticdb/LsifWriter.java

```diff
@@ -90,15 +93,21 @@ public int emitDefinitionResult(int resultSet) {
     return definitionResult;
   }
 
-  public int emitHoverResult(Semanticdb.Language language, String value) {
+  public int emitHoverResult(MarkedString... markedStrings) {
```

Comment:
It doesn't look like we use the varargs syntax when calling this method so this should be safe to change
```suggestion
  public int emitHoverResult(MarkedString[] markedStrings) {
```